src/lib/i18n/texts/de.json

```diff
@@ -66,22 +66,22 @@
       "losses": "Niederlagen",
```

Comment:
I would say so. „Siege” means “wins” and „Überzahl“ means “outnumbering the enemy”. Otherwise it would be identical to the translation of “wins” further above.